regulations/tasks.py

```diff
@@ -38,6 +47,13 @@ def build_comment(body):
 
 
 def extract_files(body):
+    '''
+    Extracts the files that are to be attached to the comment.
+    Returns a collection of dicts where for each dict:
+        - dict['key'] specifies the file to be attached from S3
+        - dict['name'] specifies the name under which the file is to be
+          attached.
+    '''
```

Comment:
Would it be a good idea to create a named tuple for this instead of passing around a dict?